pkg/capabilities/mercury/mercury.go

```diff
@@ -94,15 +108,41 @@ func (m Codec) Unwrap(raw values.Value) (ReportSet, error) {
 func (m Codec) Wrap(reportSet ReportSet) (values.Value, error) {
 	return values.NewMap(
 		map[string]any{
-			"0x1111111111111111111100000000000000000000000000000000000000000000": map[string]any{
+			feedOne.String(): map[string]any{
 				"timestamp": 42,
 				"price":     decimal.NewFromFloat(100.00),
 			},
 		},
 	)
 }
 
+// feedReport is a values-compatible version of <FeedReport>, where
+// <FeedID> is a string rather than a custom type.
+type feedReport struct {
+	FeedID               string `json:"feedId"`
```

Comment:
Given that we need FeedID to be a string for the values framework, would it make sense to just have the field be a string in the FeedReport struct and convert from byte[] to string when creating the FeedReport? I have to check where we are using it but I assume we could convert it back to byte[] for things that need it to be a byte[] too.